maintainers/maintainer-list.nix

```diff
@@ -9027,6 +9027,12 @@
     githubId = 1667473;
     name = "Jethro Kuan";
   };
+  jetpackjackson = {
+    email = "baileyannew@tutanota.com";
+    github = "JetpackJackson";
+    githubId = 88674707;
+    name = "Bailey Watkins";
+  };
```

Comment:
Alright. Since I had already edited the file, how should I best make it so that my edit to that file is a separate commit? Should I edit it again somehow and recommit it?

Reply:
Ok I'm currently trying to squash and split my various commits so that it's properly set up, but it's taking me a while, I'm sorry for the delay.